system_metrics_collector/test/system_metrics_collector/test_periodic_measurement_node.cpp

```diff
@@ -130,8 +142,17 @@ constexpr std::chrono::milliseconds PeriodicMeasurementTestFixure::kDontPublishD
 
 TEST_F(PeriodicMeasurementTestFixure, Sanity) {
   ASSERT_NE(test_periodic_measurer_, nullptr);
+
+  ASSERT_EQ("name=test_periodic_node, measurement_period=50ms,"
+    " publishing_topic=, publish_period=500ms, started=false,"
```

Comment:
An empty ```publishing_topic``` isn't indicative of what was actually set. I'm not a fan of this change. If we want to tie it to publisher state then I would only display when started.

Reply:
Okay, changed this test to call `Start()` on the measurement node beforehand.